src/python/pants/backend/helm/util_rules/chart.py

```diff
@@ -69,6 +69,10 @@ class HelmChart:
     def name(self) -> str:
         return self.info.name
 
+    @property
+    def description(self) -> str:
+        return self.name
```

Comment:
Just added this instead of writing a wrapper around `Chart` just for `description`.